conda_devenv/devenv.py

```diff
@@ -91,18 +92,21 @@ def handle_includes(root_filename, root_yaml):
             continue
 
         for included_filename in yaml_dict.get("includes") or []:
-            included_filename = os.path.abspath(included_filename)
-            if not os.path.isfile(included_filename):
+            if Path(included_filename).is_absolute():
+                included_filename = Path(included_filename)
+            else:
+                included_filename = Path(filename).parent / included_filename
+
+            if not included_filename.is_file():
                 raise ValueError(
                     "Couldn't find the file '{included_filename}' "
                     "while processing the file '{filename}'.".format(
                         included_filename=included_filename, filename=filename
                     )
                 )
-            with open(included_filename, "r") as f:
-                jinja_contents = render_jinja(
-                    f.read(), included_filename, is_included=True
-                )
+            jinja_contents = render_jinja(
+                included_filename.read_text(), included_filename, is_included=True
```

Comment:
```suggestion
                included_filename.read_text(encoding="UTF-8"), included_filename, is_included=True
```

(note that `encoding` should have been passed to the previous code too, but was an oversight)

Reply:
Done!